crates/runtime/src/jit_function_registry.rs

```diff
@@ -12,6 +12,7 @@ lazy_static! {
 pub struct JITFunctionTag {
     pub module_id: Option<String>,
     pub func_index: usize,
+    pub func_name: String,
```

Comment:
I have concerns about copying both the module and function names into this data structure for every function in a module, especially for modules with very large function counts.  I wonder if things could be refactored to store a `Rc<Module>` that could be used to get at the symbolic data when we actually need it for a trace.

Reply:
Valid concern: I did not really want global variable to hold a reference to a Module.

Reply:
Agreed, that might not be the best way to solve the problem given the lifetime of the JIT function registry data.  I don't want this comment to hold up progress on this, but a smaller footprint of this data will be eventually desired.